common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/Format.scala

```diff
@@ -98,12 +100,12 @@ object Format extends Enum[Format] {
     override val label: String = "Mixed materials"
   }
 
-  case object Audio extends Unlinked {
+  case object Audio extends Unlinked with Audiovisual {
```

Comment:
How come the sealed trait in the model > maintaining a list of AV formats in the merger?